README.md

```diff
@@ -23,9 +23,10 @@ sandbox commands:
   copyFrom <file> -> copy <file> from the algod container. Useful for offline transactions & LogicSigs plus TEAL work.
 
 algorand commands:
-  logs        -> stream algorand logs with the carpenter utility.
-  status      -> get node status.
-  goal (args) -> run goal command like 'goal node status'.
+  logs            -> stream algorand logs with the carpenter utility.
+  status          -> get node status.
+  goal (args)     -> run goal command like 'goal node status'.
+  tealdbg (args)  -> run tealdbg command like 'tealdbg help'. In sandbox, always use `tealdbg` with `--listen 0.0.0.0` option!
```

Comment:
Change the description to:
"-> run tealdbg command to debug program execution."

You can drop the listen flag notice as it is appended by default.

Reply:
Good, I changed the sentence as instructed by @ryanRfox .
In case of --listen flag, I have added this because default appended --listen flag is set to 127.0.0.1 as of listening on localhost loopback network only and that reminder notifies users to change it to 0.0.0.0 in case for listening on all container attached networks or specific container IP address that user prefers. Now considering this fact, I'm awaiting further instructions!